### PR TITLE
Added doc for formatTime helper

### PR DESCRIPTION
**Pull request by jklepatch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,9 +69,16 @@ The **Format Tags** helper takes a comma separated list of tags and render them
...
+### Format time (uses moment.js)
 
-[ToDo]
+The **Format time** helper transform a date into a user-friendly format. It uses the [moment.js library](https://momentjs.com/docs/).
+
+The `date` and `format` parameters are documented respectively [here](https://momentjs.com/docs/#/parsing/string/) and [here](https://momentjs.com/docs/#/parsing/string-format/).
+
+**Example**
+```
+{{#formatTime date format}}
```
Comment (toh82) [inline README.md]: you only need the `#` in handlebars if your helper has a body, for this, you can remove the `#`  
see http://handlebarsjs.com/block_helpers.html   
and the helper in use, within my blog: https://github.com/toh82/me/blob/master/src/partials/blogArticle.hbs

**Review by toh82 — COMMENTED** [review]
Very nice, also with linking the original source.

File: README.md (hunk 1 above)
Comment (jklepatch) [reply README.md]: Ok :)
I removed the # and added the example of your blog

**Review by toh82 — APPROVED** [review]

**Comment by toh82** [comment]
Perfect. Many thanks for your contribution on this!

**Pull request merged**